src/components/categoryPropTypes.js

```diff
@@ -0,0 +1,9 @@
+import PropTypes from 'prop-types';
+
+export default PropTypes.shape({
+    /** Name of a category */
+    name: PropTypes.string.isRequired,
+
+    /** Flag that determines if a category is active */
```

Comment:
NAB: small suggestion 
```suggestion
    /** Flag that determines if a category is active and able to be selected */
```

Reply:
Done - bcf2d431b0f889165477c34acc8ff88d86d5c270